frontend/src/container/Sidebar.jsx

```diff
@@ -33,6 +33,7 @@ export class Sidebar extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
+      activeWorkspaceId: -1,
```

Comment:
it is a good practice to explain magic numbers.
"magic numbers" are numbers in the code base that aren't self explanatory.
You can add a comment next to it or use a meaningful constant like `NO_ACTIVE_SPACE_ID` for example (if I understand correctly)

Reply:
Updated.